products/views.py

```diff
@@ -1,3 +1,59 @@
-from django.shortcuts import render
+from django.db.models       import Q
+from django.http            import JsonResponse
+from django.views           import View
 
-# Create your views here.
+from products.models import Image, ProductColor
+
+class ProductListView(View): 
+    def get(self, request): 
+        category = request.GET.getlist('category', None)
+        color= request.GET.getlist('color',None)
```

Comment:
값이 리스트로 들어올 수 있는 변수는 꼭 복수형으로 선언해주세요.

```suggestion
        categories = request.GET.getlist('category', None)
        colors = request.GET.getlist('color',None)
```